parts/k8s/cloud-init/artifacts/kubelet.service

```diff
@@ -16,15 +16,6 @@ ExecStartPre=/bin/bash -c "if [ $(mount | grep \"/var/lib/kubelet\" | wc -l) -le
 ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
 {{/* This is a partial workaround to this upstream Kubernetes issue: */}}
 {{/* https://github.com/kubernetes/kubernetes/issues/41916#issuecomment-312428731 */}}
-ExecStartPre=/sbin/sysctl -w net.ipv4.tcp_retries2=8
```

Comment:
@xizhamsft @yangl900 @xuto2 FYI, we are moving these sysctl configs into the new api model interface `sysctldConfig` (available for masterProfile and agentPoolProfiles).

No kernel parameter configuration changes, just changing how they are implemented in aks-engine.

These aks-engine changes will allow for rapid experimentation of various kernel params, no need to change code.

Reply:
@jackfrancis  Good to know. Thanks for the notice!